src/Illuminate/Foundation/Testing/AssertionsTrait.php

```diff
@@ -14,7 +14,7 @@ public function assertResponseOk()
 	{
 		$actual = $this->response->getStatusCode();
 
-		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' .$actual);
+		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' . $actual);
```

Comment:
With all due respect, are you going to reject people's PRs just because there is a bloody space in between dots? How do you recommend I put spaces around dots in the line below?! Does this project has some coding guidelines I missed? Because what you ask doesn't even relate to PSR guidelines. Correct me if I am wrong. I am bloody care about this project as much as any developer who uses it in a daily basis. Please show some understanding! :( Have a good day!